libfwupd/fwupd-enums.h

```diff
@@ -493,6 +493,15 @@ typedef enum {
  * Since: 1.7.1
  */
 #define FWUPD_DEVICE_FLAG_AFFECTS_FDE (1llu << 45)
+/**
+*
+* FWUPD_DEVICE_FLAG_NO_AUTO_REMOVE :
+*
+* The device is not auto removed.
+* Since 1.7.2
```

Comment:
This is now 1.7.3

Reply:
fixed in https://github.com/fwupd/fwupd/pull/4046/commits/02304c8e6ecd9c8484696bb959507e81b706efb8